spec/spec_helper.rb

```diff
@@ -18,78 +18,156 @@
   end
 end
 
-def fake_authenticator_data(rp_id: nil, user_present: true, credential_public_key: nil)
-  rp_id ||= "localhost"
-  rp_id_hash = OpenSSL::Digest::SHA256.digest(rp_id)
+class FakeAuthenticator
```

Comment:
How about moving things to `spec/support/fake_authenticator.rb` ?

Reply:
Good idea